src/CommunityToolkit.Maui.UnitTests/BaseTest.cs

```diff
@@ -38,4 +39,34 @@ void IDisposable.Dispose()
 		Dispose(true);
 		GC.SuppressFinalize(this);
 	}
+
+	protected static Stream GetStreamFromImageSource(ImageSource imageSource)
+	{
+		var streamImageSource = (StreamImageSource)imageSource;
+
+		var cancellationToken = System.Threading.CancellationToken.None;
+		var task = streamImageSource.Stream(cancellationToken);
+		return task.Result;
+	}
+
+	protected static bool StreamEquals(Stream a, Stream b)
+	{
+		if (a == b)
+			return true;
+
+		if (a == null
+			|| b == null
```

Comment:
You can get rid of the `null` checks since neither `Stream a`, nor `Stream b` are nullable

Reply:
Got it - done